ossfuzz/fuzz_data_producer.h

```diff
@@ -3,40 +3,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#include "fuzz_helpers.h"
+
 typedef struct {
```

Comment:
Then here, you can just do:

```c
typedef struct FUZZ_dataProducer_s FUZZ_dataProducer_t;
```

This is how we hide struct internals in C where we don't have private members.